server/src/main/java/org/apache/uniffle/server/storage/LocalStorageManager.java

```diff
@@ -113,22 +114,27 @@ public class LocalStorageManager extends SingleStorageManager {
     }
     ExecutorService executorService = ThreadUtils.getDaemonCachedThreadPool("LocalStorage-check");
     LocalStorage[] localStorageArray = new LocalStorage[storageBasePaths.size()];
+    boolean isDiskCapacityWatermarkCheckEnabled = conf.get(DISK_CAPACITY_WATERMARK_CHECK_ENABLED);
     for (int i = 0; i < storageBasePaths.size(); i++) {
       final int idx = i;
       String storagePath = storageBasePaths.get(i);
       executorService.submit(
           () -> {
             try {
               StorageMedia storageType = getStorageTypeForBasePath(storagePath);
-              localStorageArray[idx] =
+              LocalStorage.Builder builder =
                   LocalStorage.newBuilder()
                       .basePath(storagePath)
                       .capacity(capacity)
                       .ratio(ratio)
                       .lowWaterMarkOfWrite(lowWaterMarkOfWrite)
                       .highWaterMarkOfWrite(highWaterMarkOfWrite)
                       .localStorageMedia(storageType)
-                      .build();
+                      .enableDiskCapacityWatermarkCheck();
```

Comment:
It does not look like this line is intended?

Reply:
My fault. It has been removed.